context/shared/src/main/scala-2.12_2.13/com/karasiq/bootstrap/context/TextClassModifiers.scala

```diff
@@ -0,0 +1,57 @@
+package com.karasiq.bootstrap.context
+
+import scalatags.Escaping
+import scalatags.text.Builder.ValueSource
+
+import scala.language.postfixOps
+
+trait TextClassModifiers extends ClassModifiers { self: TextRenderingContext ⇒
+  def addClass(element: Element, className: String): Unit = {
+    modifyClasses(element, _ + className)
+  }
+
+  def removeClass(element: Element, className: String): Unit = {
+    modifyClasses(element, _ - className)
+  }
+
+  private[this] final case class ClassesValueSource(classes: Set[String]) extends ValueSource {
+    def appendAttrValue(strb: java.io.Writer): Unit = {
+      if (classes.nonEmpty) {
+        val iterator = classes.iterator
+        Escaping.escape(iterator.next(), strb)
```

Comment:
scalatags 0.9.2 requires the second param of `Escaping.escape` to be `java.io.Writer` whereas it's `StringBuilder` for 0.6.8 .